src/servers/src/http.rs

```diff
@@ -708,10 +709,24 @@ impl HttpServer {
                         authorize::check_http_auth,
                     )),
             )
+            .nest(
+                &format!("/{HTTP_API_VERSION}/admin"),
+                Router::new()
+                    // handler for changing log level dynamically
+                    .route(
+                        "/log_level",
+                        routing::get(dyn_log::dyn_log_handler).post(dyn_log::dyn_log_handler),
+                    ),
+            )
```

Comment:
```suggestion
```

Let's not add an `/admin` router here. Putting it under `/debug/log_level` would be enough.